code/modules/recycling/disposal-construction.dm

```diff
@@ -224,9 +220,7 @@
 			anchored = 1
 			if(ispipe)
 				level = 1 // We don't want disposal bins to disappear under the floors
-				density = 0
-			else
-				density = 1 // We don't want disposal bins or outlets to go density 0
+			density = 0

```

Comment:
Are you sure this doesn't make disposal bin not dense like the comment above says?